moveit_kinematics/kinematics_base_test/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
We can't have catkin package inside of a catkin package... moveit_kinematics is already a package

I think this subfolder should not be a ROS package but instead just a subfolder of moveit_kinematics